packages/create-sitecore-jss/src/templates/nextjs/src/middleware.ts

```diff
@@ -14,7 +14,6 @@ export const config = {
    * 3. /sitecore/api (Sitecore API routes)
    * 4. /- (Sitecore media)
    * 5. /healthz (Health check)
-   * 6. all root files inside /public (e.g. /favicon.ico)
    */
-  matcher: ['/', '/((?!api/|_next/|healthz|sitecore/api/|-/|[\\w-]+\\.\\w+).*)'],
+  matcher: ['/', '/((?!api/|_next/|healthz|sitecore/api/|-/).*)'],
```

Comment:
Should we keep exclusion of /public files we ship by default (favicon.ico and sc_logo.svg) as an example - just make it explicit? It's probably a worthy optimization to have and to point out.

Reply:
Good idea, it will not hurt for sure :)